classes/Hotel.js

```diff
@@ -29,7 +29,7 @@ calculateTotalCostOfAllCustomerBookings(customer) {
     })
     return acc;
   }, 0);
-this.totalBookingCost = Number(customerTotalBookingCost.toFixed(2));
+    this.totalBookingCost = Number(customerTotalBookingCost.toFixed(2));
```

Comment:
Great use of using a JS feature to found the price to a legible number (I know how many decimal numbers you'd get elsewise!)